### DISCUSSION TITLE
Trigger doorbell via second hardware button

### DISCUSSION THREAD

**Discussion by nicx** [koush/scrypted#755] [Q&A] [answered]
I am using a Unifi G4 Pro doorbell in combination with HomeKit. Works great! :)

Now i want to trigger the doorbell via an additional hardware button. for this i have a shelly uni connected to this button. i now see 2 possibilities:

1. either the shelly signals the pressing of the switch via webhook directly to the doorbell.
2. or i detect the status of the shelly in HomeKit, and in HomeKit i build an automation to press the doorbell

For both I need some "instance" in Scrypted to trigger the doorbell virtually.

What I did so far:
I tried to activate and trigger the OnOff-Webhook directly in the Doorbell Unifi Device in Scrypted, but nothing happens. Is this to be expected?
I also tried to add a "Custom Doorbell Button" via the Dummy Switch Plugin... but I have no glue where this custom button should appear?!

Probably it would work if I configure a DummySwitch with Webbhook, then put it together with the Doorbell in a group, and then use it e.g. in HomeKit. But that seems rather cumbersome to me.... is there a way to trigger the doorbell directly without detour via a group from the outside?

**Comment by koush** [comment, marked as answer]
Using the new dummy switch replace doorbell extension or a device group are your options.
But for your use case, it's only device group if you want two buttons.

**Comment by nicx** [comment]
@koush thanks, but I dont understand your answer.

What exactly is needed to have another extra psysical button pressing virtually the existing doorbell?
